setup.cfg

```diff
@@ -0,0 +1,2 @@
+[flake8]
```

Comment:
I'm thinking we should implement a sort of github actions thingy enforcing these things